### ISSUE TITLE
Black on black feed text is unreadable

### ISSUE THREAD

**Issue by adamzea** [lposidon/posidonLauncher#74]
The feed shows headlines with black text on top of the feed images. This makes the headlines completely unreadable if there is any contrast in the image and the image contains anything other than a solid light color. 
![Screenshot_20201220-124330](https://user-images.githubusercontent.com/24486391/102720352-d2577380-42c1-11eb-9e9b-d52f677fd592.png)

**Comment by thealamu** [comment]
Have you played with the colors in the news settings? What happens when you set the text color to white for example?

**Comment by adamzea** [comment]
> Have you played with the colors in the news settings? What happens when you set the text color to white for example?

Yes.  Then the text becomes unreadable when it's on top of a light colored featured image... white on white. Or parts of the text become unreadable over light parts of the background image. 

The standard color contrast ratio for text on a background image is 4.5:1.  See: https://uxdesign.cc/the-easiest-part-about-designing-accessible-websites-76cd6b9a7ae4   When you layer text over continuous tone images, that contrast ratio is all over the place which makes the text very hard to read.  A better way would be to either drastically reduce the contrast within the background image with an overlay or blur filter in order to increase the contrast between the text and background, or modify the layout so that the title text appears on a different background that provides more contrast than the variable images contained in the RSS feeds.

**Comment by thealamu** [comment]
I think there is an option to move the text under the background image but I also agree the blur filter is a better and more 
aesthetically pleasing way.
PS: Your link seems to be broken for me, I'd love to give it a read.
